src/js/adjustHeights.js

```diff
@@ -1,27 +1,42 @@
 import $ from 'jquery';
 import _ from 'lodash';
 
-const setHeightsArray = [
-  // { className: 'flex-col--2__h1', parentId: 'naprawa'},
-  // { className: 'flex-col--2__p--1', parentId: 'naprawa'},
-  // { className: 'flex-col--2__p--2', parentId: 'naprawa'},
-  // { className: 'flex-col--2__list', parentId: 'naprawa'},
+const setHeightsArrayDesktop = [
+  { className: 'section__header', parentId: 'naprawa' },
+  { className: 'section__p--1', parentId: 'naprawa' },
+  { className: 'section__p--2', parentId: 'naprawa' },
+  { className: 'section__list-div', parentId: 'naprawa' },
+  { className: 'section__header', parentId: 'produkcja' },
+  { className: 'section__p--1', parentId: 'produkcja' },
+  { className: 'section__p--2', parentId: 'produkcja' },
+  { className: 'section__list-div', parentId: 'produkcja' }
```

Comment:
Please change name of `className` to `element` and `parentId` to `parent`.
I see some inconsistency - once you use `class` and then you use `id` to refer to css class of element

- [ ] inconsistency of naming

Reply:
>  I can rename className to elementClass to be more consistent, though.
Great!